modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/org.elasticsearch.txt

```diff
@@ -148,6 +148,13 @@ class org.elasticsearch.index.fielddata.ScriptDocValues$Doubles {
   double getValue()
 }
 
+class org.elasticsearch.index.fielddata.ScriptDocValues$Geometry {
+        double getCentroidLat();
+        double getCentroidLon();
+        double width();
+        double height();
```

Comment:
I think these name might be confusing since these are not width and height of the shape but rather width and height of the bounding box. Which makes me think maybe we should return a point and a bounding box here. In case of the bounding box we will also give access to min/max lat/lon which we have to calculate anyway.